src/Common/tests/System/Net/Capability.Sockets.cs

```diff
@@ -47,5 +47,10 @@ public static bool IPv4Support()
         {
             return Socket.OSSupportsIPv4;
         }
+
+        public static bool UnixDomainSocketsSupport()
+        {
+            return !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
It would be internal on SocketProtocolSupportPal.Unix but then it would have to be a new public static method on Socket for common\tests\system\net\capability.sockets.cs to use it. So it's a no go then. I'll remove UnixDomainSocketsSupport()